muxers/yamux/src/lib.rs

```diff
@@ -231,7 +231,9 @@ impl WindowUpdateMode {
     /// > size must be tuned appropriately for the desired
     /// > throughput and level of tolerance for (temporarily)
     /// > slow receivers.
+    #[deprecated(note = "Use `WindowUpdateMode::on_read` instead.")]
```

Comment:
What is the point of having the entire `WindowUpdateMode` struct if it has only one option? Shouldn't it be deprecated entirely?